### PR TITLE
fix: Return correct GPG keys url

### PR DESCRIPTION
**Pull request by Ushie**

### CODE REVIEW

**Review by oSumAtrIX — COMMENTED** [review]
This is difficult because while GitHub as a backend could support this format, other backends don't necessarily. The .gpg is the universal format from which other attributes like the fingerprint or key id can be read. Maybe for the use of website, only the link to the gpg is shown and no fingerprint or key id?

**Comment by Ushie** [comment]
Regardless of the solution you decide to go with, this is already present in the API and can't (shouldn't) be removed, it would be best to fix the response

**Comment by Ushie** [comment]
It currently 404s
image

**Comment by oSumAtrIX** [comment]
Sounds weird that casing matters. I think .gpg.is appended to the GitHub username, but it looks like the display name is used

**Comment by Ushie** [comment]
It's not the casing, look at the domain

File: src/main/kotlin/app/revanced/api/configuration/repository/GitHubBackendRepository.kt [hunk 1]
```diff
@@ -98,7 +98,7 @@ class GitHubBackendRepository(client: HttpClient) : BackendRepository(client) {
                 gpgKeys =
                 BackendMember.GpgKeys(
                     ids = gpgKeys.map { it.keyId },
-                    url = "https://api.github.com/users/${user.login}.gpg",
+                    url = "https://api.github.com/users/${user.login}/gpg_keys",
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/api/configuration/repository/GitHubBackendRepository.kt:101]:
```suggestion
                    url = "https://github.com/${user.login}.gpg",
```

**Pull request merged**